pkgs/os-specific/linux/kernel/manual-config.nix

```diff
@@ -309,6 +309,8 @@ stdenv.mkDerivation ((drvAttrs config stdenv.hostPlatform.linux-kernel kernelPat
   pname = "linux";
   inherit version;
 
+  pos = builtins.unsafeGetAttrPos "version" args;
+
```

Comment:
Not sure if I'm helpful, but I was thinking you could let the caller set `pos`.

You either want to require `pos` or use `optionalAttrs` to set it only when it's actually passed: `// optionalAttrs args?pos { inherit (args) pos; }` and then allow it in the arg list.

Reply:
Done